examples/latex.dx

```diff
@@ -30,14 +30,22 @@
 
 '## No LaTeX rendering in non-prose-blocks
 
-'LaTeX rendering should not occur in code blocks, not in error output cells.
+'LaTeX rendering should not occur in code blocks, nor in error output cells.
 
-def arraySum (x:a=>Int32) : Int32 =
-  initAcc = 0
+def array_sum {a} (x:a=>Int32) : Int32 =
   -- Note: the following line has `$ ... $`, but it should not trigger KaTeX.
-  snd $ with_state initAcc $ \acc.
+  -- Note: the incorrect usage of `with_state` below is intentional to verify
+  --       that `$ ... $` is not rendered as LaTeX in error output cells.
+  snd $ with_state 0 $ \acc.
```

Comment:
This is a legit type error and we shouldn't commit the code like that. Either use `snd $ run_state`, or better `yield_state`.

Reply:
Yes, this is a legit type error.

The purpose of this test is to verify that `$ ... $` in error output cells does not display as LaTeX, so the error is intentional – see https://github.com/google-research/dex-lang/pull/447 for context.

If tests pass, I think things are working as intended.

Reply:
Ah, sure. I missed the comment 🤦